Userland/Libraries/LibWeb/HTML/Scripting/Intrinsics.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022, Andrew Kaster <akaster@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/HashMap.h>
+#include <AK/String.h>
+#include <LibJS/Forward.h>
+#include <LibJS/Runtime/NativeFunction.h>
+#include <LibJS/Runtime/Object.h>
+#include <LibWeb/HTML/Scripting/Intrinsics.h>
+
+namespace Web::HTML {
+
+JS::Object& Intrinsics::cached_web_prototype(String const& class_name)
```

Comment:
You're gonna hate me for this, but I don't think this belongs in HTML/Scripting/ - things from https://html.spec.whatwg.org/multipage/scripting.html belong there, but most non-standard stuff (like MainThreadVM belongs in Bindings/.

Reply:
El moved. Wasn't too painful doing this at the same time I made sure each commit built with the power of find + sed.